src/components/ProductListItem.tsx

```diff
@@ -0,0 +1,48 @@
+"use client";
+import Stripe from "stripe";
+import { useRouter } from "next/navigation";
+import { setProduct } from "@/redux/features/cartSlice";
+import React from "react";
+import { useAppDispatch } from "@/redux/store";
+
+type ProductListItemProps = {
+  item: Stripe.Price;
+};
+
+const ProductListItem: React.FC<ProductListItemProps> = ({ item }) => {
+  const { id, unit_amount, product } = item;
```

Comment:
Property 'description' does not exist on type 'string | Product | DeletedProduct'.